MinecraftClient/Resources/lang/de.ini

```diff
@@ -0,0 +1,474 @@
+[mcc]
+# Messages from MCC itself
+mcc.login=Login :
+mcc.login_basic_io=Bitte gebe einen Nutzernamen oder eine E-Mail deiner Wahl ein.
+mcc.password=Passwort : 
+mcc.password_basic_io=Bitte gebe das Passwort für {0} ein.
+mcc.password_hidden=Passwort : {0}
+mcc.offline=§8Das Programm läuft im offline Modus.
+mcc.session_invalid=§8Gespeicherte Session ungültig oder abgelaufen.
+mcc.session_valid=§8Gespeicherte Session gültig für {0}.
+mcc.connecting=Verbinde zu {0}...
+mcc.ip=Server IP : 
+mcc.use_version=§8Benutze Minecraft Version {0} (protocol v{1})
+mcc.unknown_version=§8Unbekannte oder nicht unterstützte Minecraft Version {0}.\nWechsele in den autodetection Modus.
+mcc.forge=Prüfe ob der Server Forge benutzt...
+mcc.forgeforce=Zwang-Aktivierung des Forge supports.
+mcc.resolve=Löse auf {0}...
+mcc.found=§8Server gefunden {0}:{1} für Adresse {2}
+mcc.not_found=§8SRV lookup fehlgeschlagen für {0}\n{1}: {2}
+mcc.retrieve=Erhalte Server Info...
+mcc.restart=Starte Minecraft Console Client neu...
+mcc.restart_delay=Warte {0} Sekunden vor Neustart...
+mcc.server_version=Server Version : 
+mcc.disconnected=Mit keinem Server verbunden. Schreibe '{0}help' für weitere Hilfe.
+mcc.press_exit=Oder drücke Enter um den Minecraft Console Client zu verlassen.
+mcc.version_supported=Version wird unterstützt.\nMelde an...
+mcc.single_cmd=§7Befehl §8 {0} §7 gesendet.
+mcc.joined=Server erfolgreich beigetreten.\nTippe '{0}quit', um ihn wieder zu verlassen.
+mcc.reconnect=Warte 5 Sekunden ({0} Versuche übrig)...
+mcc.disconnect.lost=Verbindung wurde verloren.
+mcc.disconnect.server=Server hat die Verbindung abgebrochen :
+mcc.disconnect.login=Login fehlgeschlagen :
+mcc.link=Link: {0}
+mcc.player_dead_respawn=Du bist gestorben. Automatischer Respawn nach 1 Sekunde.
+mcc.player_dead=Du bist gestorben. Schreibe /respawn zum Respawnen.
+mcc.server_offline=§8Der Server läuft im offline Modus.
+mcc.session=Überprüfe Session...
+mcc.session_fail=Konnte Session nicht überprüfen.
+mcc.server_protocol=§8Server Version : {0} (protocol v{1})
```

Comment:
```suggestion
mcc.server_protocol=§8Server-Version : {0} (protocol v{1})
```

Reply:
https://context.reverso.net/%C3%BCbersetzung/englisch-deutsch/the+server+version